source/datasets/bikesharing.rst

```diff
@@ -4,7 +4,7 @@
 Output                     JSON, mime-type application/json
 E-mail contact             |contact|
 API version                :strike:`v1` |deprecated|, v2
-Swagger URL                |apiv2|
+Swagger URL                https://mobility.api.opendatahub.bz.it/v2/flat/Bicycle,BikesharingStation,Bicyclestationbay
```

Comment:
We should not call it "Swagger URL" anymore, since it does point directly to the browsable API